flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDynamicTableFactoryTest.java

```diff
@@ -66,6 +71,36 @@ public static void setup() {
         tableEnv.getConfig().setSqlDialect(SqlDialect.HIVE);
     }
 
+    @Test
+    public void testHiveTableSourceJobConfWithCredentials() throws Exception {
```

Comment:
We inject the token for both source and sink. So let's test for both of them too.

Reply:
Done